<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetBrowserDialog</class>
 <widget class="QMainWindow" name="AssetBrowserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asset Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0">
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <item row="0" column="0" colspan="3">
     <widget class="CAssetBrowserSearchDlg" name="m_searchDlg" native="true"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="CAssetViewer" name="m_assetViewer" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="m_statusBar0">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="m_statusBar1">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>738</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>AssetBrowserToolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBuildCache"/>
   <addaction name="actionShowList"/>
   <addaction name="actionShowPreview"/>
   <addaction name="actionShowFilters"/>
   <addaction name="separator"/>
   <addaction name="actionSelectInViewport"/>
   <addaction name="actionSelectFromViewport"/>
  </widget>
  <action name="actionBuildCache">
   <property name="icon">
    <iconset resource="AssetBrowser.qrc">
     <normaloff>:/AssetBrowser/ab_toolbar_00.png</normaloff>:/AssetBrowser/ab_toolbar_00.png</iconset>
   </property>
   <property name="text">
    <string>Cache info and thumbnails</string>
   </property>
   <property name="toolTip">
    <string>Cache info and thumbnails</string>
   </property>
  </action>
  <action name="actionShowList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AssetBrowser.qrc">
     <normaloff>:/AssetBrowser/ab_toolbar_01.png</normaloff>:/AssetBrowser/ab_toolbar_01.png</iconset>
   </property>
   <property name="text">
    <string>Show List of Assets</string>
   </property>
   <property name="toolTip">
    <string>Show list of Assets including Path, LOD's, Triangles, Submeshes, Materials, Tags</string>
   </property>
  </action>
  <action name="actionShowFilters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AssetBrowser.qrc">
     <normaloff>:/AssetBrowser/ab_toolbar_03.png</normaloff>:/AssetBrowser/ab_toolbar_03.png</iconset>
   </property>
   <property name="text">
    <string>Show Filter Options</string>
   </property>
   <property name="toolTip">
    <string>Show Filter Options</string>
   </property>
  </action>
  <action name="actionShowPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="AssetBrowser.qrc">
     <normaloff>:/AssetBrowser/ab_toolbar_02.png</normaloff>:/AssetBrowser/ab_toolbar_02.png</iconset>
   </property>
   <property name="text">
    <string>Show Preview of Selected Asset</string>
   </property>
   <property name="toolTip">
    <string>Show Preview of Selected Asset</string>
   </property>
  </action>
  <action name="actionSelectInViewport">
   <property name="icon">
    <iconset resource="AssetBrowser.qrc">
     <normaloff>:/AssetBrowser/ab_toolbar_06.png</normaloff>:/AssetBrowser/ab_toolbar_06.png</iconset>
   </property>
   <property name="text">
    <string>Select objects in viewport using the selected assets</string>
   </property>
  </action>
  <action name="actionSelectFromViewport">
   <property name="icon">
    <iconset resource="AssetBrowser.qrc">
     <normaloff>:/AssetBrowser/ab_toolbar_07.png</normaloff>:/AssetBrowser/ab_toolbar_07.png</iconset>
   </property>
   <property name="text">
    <string>Select assets used by the viewport selected objects</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CAssetBrowserSearchDlg</class>
   <extends>QWidget</extends>
   <header>AssetBrowser/AssetBrowserSearchDlg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CAssetViewer</class>
   <extends>QWidget</extends>
   <header>Controls/AssetViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="AssetBrowser.qrc"/>
 </resources>
 <connections/>
</ui>
